aiida/cmdline/params/options/interactive.py

```diff
@@ -209,7 +219,7 @@ def simple_prompt_loop(self, ctx, param, value):
                     return value
 
     def after_callback(self, ctx, param, value):
-        """If a callback was registered on init, call it and return it's value"""
+        """If a callback was registered on init, call it and return it's value."""
```

Comment:
It should have been `its` instead of `it's`... anyway we can fix this in another PR